codec/src/main/java/io/netty/handler/codec/marshalling/MarshallingDecoder.java

```diff
@@ -81,7 +81,7 @@ protected Object decode(ChannelHandlerContext ctx, ByteBuf in) throws Exception
     }
 
     @Override
-    protected ByteBuf extractFrame(ChannelHandlerContext ctx, ByteBuf buffer, int index, int length) {
+    protected ByteBuf extractFrame(ByteBuf buffer, int index, int length) {
```

Comment:
And remove the method `extractFrame(ByteBuf buffer, int index, int length)` from `MarshallingDecoder`. It's present in the `BaseLengthBasedFrameDecoder`.